mods/cnc/chrome.yaml

```diff
@@ -146,31 +145,6 @@ textfield-focused: chrome.png
 	corner-bl: 64,254,2,2
 	corner-br: 126,254,2,2
 
-# A copy of button
```

Comment:
Is this deliberate?  Progress bars are used in more places than just the installer!